package.json

```diff
@@ -83,56 +81,39 @@
     "@vue/vue2-jest": "^27.0.0-alpha.4",
     "babel-jest": "^27.4.5",
     "babel-loader": "8.2.3",
+    "browserslist": "^4.19.1",
     "chart.js": "^2.8.0",
     "commitizen": "^4.2.4",
-    "connect-history-api-fallback": "^1.1.0",
     "cross-env": "^5.1.1",
-    "cross-spawn": "^5.1.0",
     "css-loader": "^0.28.0",
     "cz-conventional-changelog": "3.3.0",
     "eslint": "7.28.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-config-standard": "^10.2.1",
-    "eslint-friendly-formatter": "^2.0.7",
     "eslint-loader": "^4.0.2",
-    "eslint-plugin-html": "^2.0.1",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-node": "^4.2.2",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-prettier-vue": "^3.1.0",
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-standard": "^3.0.1",
     "eslint-plugin-vue": "^8.2.0",
-    "eventsource-polyfill": "^0.9.6",
-    "express": "^4.15.2",
-    "extract-text-webpack-plugin": "^3.0.1",
-    "file-loader": "^0.10.1",
-    "friendly-errors-webpack-plugin": "^1.6.1",
-    "function-bind": "^1.0.2",
-    "html-webpack-plugin": "^2.28.0",
-    "http-proxy-middleware": "^0.17.4",
-    "inject-loader": "^3.0.0",
     "jest": "^27.4.3",
     "jest-canvas-mock": "^2.3.1",
-    "json-loader": "^0.5.4",
     "lint-staged": "^12.1.2",
-    "opn": "^5.1.0",
-    "portfinder": "^1.0.13",
     "prettier": "2.5.1",
-    "shelljs": "^0.7.7",
+    "rimraf": "^3.0.2",
+    "rollup": "^2.66.1",
+    "rollup-plugin-swc": "^0.2.0",
```

Comment:
`@swc/core` is peer dependency, where is it?

alos `@swc/helpers` ?

Reply:
`@swc/core` was added in  https://github.com/apertureless/vue-chartjs/pull/733 (https://github.com/apertureless/vue-chartjs/pull/733/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R77)   
add `@swc/helpers`